### PR TITLE
Focus database projects view after open/import

### PR DESCRIPTION
**Pull request by Benjin**

### CODE REVIEW

**Comment by udeeshagautam** [comment]
Any idea why PR builds are failing... doesn't seem change related

File: extensions/sql-database-projects/src/controllers/mainController.ts [hunk 1]
```diff
@@ -89,6 +91,8 @@ export default class MainController implements Disposable {
 				for (const file of files) {
 					await this.projectsController.openProject(file);
 				}
+
+				this.projectsController.focusProject(this.projectsController.projects[0]);
```
Comment (udeeshagautam) [inline extensions/sql-database-projects/src/controllers/mainController.ts:95]: always [0]? what happens for the second project?

File: extensions/sql-database-projects/src/controllers/databaseProjectTreeViewProvider.ts [hunk 2]
```diff
@@ -58,4 +59,20 @@ export class SqlDatabaseProjectTreeViewProvider implements vscode.TreeDataProvid
 		this.roots = newRoots;
 		this._onDidChangeTreeData.fire(undefined);
 	}
+
+	public setTreeView(value: vscode.TreeView<BaseProjectTreeItem>) {
+		if (this.treeView) {
+			throw new Error('TreeView should not be set multiple times.');
```
Comment (alanrenmsft) [inline extensions/sql-database-projects/src/controllers/databaseProjectTreeViewProvider.ts:65]: it is feasible to move this to the constructor?

File: extensions/sql-database-projects/src/controllers/mainController.ts (hunk 1 above)
Comment (Benjin) [reply extensions/sql-database-projects/src/controllers/mainController.ts:95]: Good catch; meant the first of the newly-opened projects

File: extensions/sql-database-projects/src/controllers/databaseProjectTreeViewProvider.ts (hunk 2 above)
Comment (Benjin) [reply extensions/sql-database-projects/src/controllers/databaseProjectTreeViewProvider.ts:65]: Don't think so; there's a chicken-and-egg problem (check mainController.ts:68).

Basically, the treeView, understandably, requires a treeViewProvider as part of its constuctor, but I also want my treeViewProvider to be able to direct the treeView to focus on something in it.

If it were possible to get the treeViewProvider's parent treeView, then this wouldn't be an issue, so I think that's a bit of an oversight on vscode's side.

**Comment by Benjin** [comment]
> Any idea why PR builds are failing... doesn't seem change related

Looks like the build agent died.  I think I saw Amir send out an email about this where he's working with the correct team's on-call.

**Comment by alanrenmsft** [comment]
@Benjin there was an outage earlier, but it has been resolved, the PR build should be passing now.

**Comment by Benjin** [comment]
Charles confirmed that the final check (code coverage failing to upload) is related to some changes he has in flight.

**Pull request merged**